src/pages/NewsDetail.tsx

```diff
@@ -21,7 +22,27 @@ const NewsDetail = () => {
 
   return (
     <Container>
-      <AnimationWrapper>{animation}</AnimationWrapper>
+      <AnimationWrapper>
+        {animation}
+        <ScrollIndicator>
+          <span>아래로 스크롤 하여 뉴스 기사를 확인하세요</span>
```

Comment:
```suggestion
          <span>Scroll down for the article.</span>
```

사이트 다 영어니까 이것도 영어로 하는거 어때요!?